style/preview.less

```diff
@@ -352,3 +355,23 @@
 	flex-direction: column;
 	justify-content: center;
 }
+
+@media (prefers-color-scheme: dark) {
```

Comment:
Please test by setting your device or browser to dark mode, do you see a dark preview when opening the english demo page for example?

Reply:
what if ppl set to `preferColorScheme: light` when device settings to dark mode? it seems this media will overwrite the light scheme. (no .light css was given in css)

Reply:
That's a good question, the test I have found so far to answer it is using the scheme option with the rendering tab: https://developer.chrome.com/docs/devtools/rendering/emulate-css#emulate_css_media_feature_prefers-color-scheme 

Currently the media query overwrites the light scheme but we can work around that, though your question does open up another one regarding precedence: if we wish to support both the media query and an explicit/direct way to configure, which one should have more priority?

Reply:
the usage of prefersColorScheme during init() is like a "theme" to fit site owner theme, so I think prefersColorScheme has more priority than the system dark mode setting.

we are here mix the "theme" and dark mode, would be nice to use css class name like "theme-dark" to differentiate them, although we share the same styling.

Reply:
Updated as discussed, please test carefully